docs/reference.md

```diff
@@ -430,6 +433,21 @@ OpenWhisk has a few system limits, including how much memory an action uses and
 * A user cannot change this limit when creating the action.
 * A CLI call that exceeds this limit receives an error code corresponding to TOO_MANY_REQUESTS.
 
+### Size of the code per action (Fixed: 48MB)
```

Comment:
There is a bug/feature (take your pick) in the nodejs proxy which limits the body payload to 1MB - this should be removed: https://github.com/openwhisk/openwhisk/blob/master/core/nodejsActionBase/app.js#L36 as it conflates the size of the exec and arguments unnecessarily.